listener.go

```diff
@@ -362,13 +369,36 @@ func getPkeysForTable(tableName string) ([]string, error) {
 	return columnNames, nil
 }
 
-func buildUpdateSql(tableName string, orderedColumns []string, row common.Row, pkeys []string) string {
+// Syntax "DELETE FROM Obj WHERE key1=$1 AND key2=$2 ... ;"
+func buildDeleteSql(tableName string, row common.Row, pkeys []string) string {
```

Comment:
this should be deleted.  The only version of `buildUpdateSql` should be the one that accepts the ordered columns.  Looks like a merging error.  Check out the `generic-update` branch for how this method should look.